contents/handbook/engineering/development-process.md

```diff
@@ -15,18 +15,22 @@ However, if we have no checks in place, we will release quickly but everything w
 
 ## 1. How to decide what to build
 
-There are 3 places that work comes from.
+Deciding what we work on happens in our sprint planning session, where the high level themes for the next sprint get decided. Each team then takes those high level priorities and breaks them down, plus adds their own smaller items like bugfixes, smaller features or other improvements.
 
-- Issues/bugs (raised by the community or us)
-- [Roadmap](/handbook/strategy/roadmap)
-- "This should be better"
+As one of our values is [stepping on toes](/handbook/company/values#step-on-toes), during the sprint you might come across something that should be much higher priority than what was already planned. It's up to you to then decide to work on that as opposed to what was agreed in the sprint planning session.
 
+## 2. Sizing a task (or 1-day PRs)
 
-## 2. Sizing a task
+Your PRs should be sized to be doable in one day, including code review and QA. If it's not, you need to break it down into smaller chunks until it fits into a day. Tasks of this size are easy to test, easy to deploy, easy to review and less likely to cause merge conflicts. 
 
-When picking up a task, it should be doable in a day, including code review and QA. If it's not, you need to break it down into smaller chunks until it is. Tasks of this size are easy to test, easy to deploy, less likely to cause merge conflicts, and should still deliver some kind of value.
+These PRs should still offer a slice of value, even if that value is hidden behind a feature flag for the moment.
+
+| The wrong way to do this | The right way to do this |
+| --- | --- |
+| Do backend code in one PR, then frontend in the next | Create the smallest version of the feature/bug fix that adds value |
```

Comment:
I've found that there are certain scenarios where working on the backend on one PR and on the frontend in another one is a natural good separation. Usually because the changes are so big or convoluted that it's the best way to do it safely (e.g. SAML support couldn't have been broken down in small functionality pieces properly)